lib/Wikia/src/Service/User/Permissions/PermissionsService.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+interface PermissionsService {
```

Comment:
You can still have a few fluent-ish entry points like `->user($user)` and `->as($user)` and so on but I can see you're not a great supporter of this idea. Then let's leave it as it is.